jobserver/opencodelists.py

```diff
@@ -0,0 +1,78 @@
+import requests
```

Comment:
This should probably go in the top level services directory or into the interactive directory, rather than the jobserver one.

Reply:
Services is for wrapping external services when there's config or primitives to be gained from them, rather than API wrappers.  (Slack is the odd-one-out there!)  Putting it in interactive is a great shout though 👍